src/app/pages/subjects/subjects.js

```diff
@@ -0,0 +1,167 @@
+import CategorySection from './category-section/category-section';
+
+function makeBook(title, categories) {
+    return {
+        title,
+        categories,
+        coverUrl: `https://placeholdit.imgix.net/~text?txtsize=33&txt=${title}&w=200&h=180`
+    };
+}
+
+let jsonData = () => (
+    {
+        'id': 5,
+        'meta': {
+            'type': 'books.BookIndex',
+            'detail_url': 'http://oscms-dev.openstax.org/api/v1/pages/5/'
+        },
+        'parent': {
+            'id': 3,
+            'meta': {
+                'type': 'pages.HomePage',
+                'detail_url': 'http://oscms-dev.openstax.org/api/v1/pages/3/'
+            }
+        },
+        'title': 'OpenStax Books',
+        'page_description': `<h1>Open source. Peer-reviewed. 100% free.</h1>
+         <p>And backed by additional learning resources. Review our OpenStax textbooks
+          and decide if they’re right for your course. Simple to adopt, free to use.
+          We make it easy to improve student access to higher education.</p>`,
+        'dev_standards_heading': 'Development Standards',
+        'dev_standard_1_heading': 'Content Development',
+        'dev_standard_1_description': `<p>Our open source textbooks are written by
+         professional content developers who are experts in their fields.</p>`,
+        'dev_standard_2_heading': 'Standard Organization',
+        'dev_standard_2_description': `<p>All textbooks meet standard scope and
+         sequence requirements, making them seamlessly adaptable into existing
+         courses.</p>`,
+        'dev_standard_3_heading': 'Peer Review',
+        'dev_standard_3_description': `<p>OpenStax textbooks undergo a rigorous
+         peer review process. You can view the list of contributors when you
+         click on each book.</p>`
+    }
+);
+
+let dummyData = () => {
+    let categories = [
+            'Math', 'Science', 'Social Sciences', 'History', 'AP'
+        ],
+        filterButtons = ['View All', ...categories],
+        books = [
+            makeBook('Math 1', ['Math']),
+            makeBook('Math AP', ['Math', 'AP']),
+            makeBook('Math 2', ['Math']),
+            makeBook('Math 3', ['Math']),
+            makeBook('History 1', ['History']),
+            makeBook('History 2', ['History']),
+            makeBook('Science 1', ['Science']),
+            makeBook('Science 2', ['Science']),
+            makeBook('Science AP', ['Science', 'AP']),
+            makeBook('Social', ['Social Sciences'])
+        ],
+        booksByCategory = categories.map((category) => (
+            {
+                category,
+                books: books.filter((aBook) => aBook.categories.indexOf(category) >= 0)
+                    .map((book) => ({
+                        title: book.title,
+                        coverUrl: book.coverUrl
+                    }))
+            }
+        ));
+
+    return {
+        filterButtons,
+        booksByCategory
+    };
+};
```

Comment:
Since this is basically a Controller, it might be good to separate out the actual Model into a separate file to maintain a MVC-like structure.  It's possible (likely?) that this data will be relevant to other parts of the site as well.